packages/design-tokens/src/utils/typography/sanitizeFont.test.ts

```diff
@@ -43,6 +43,18 @@ describe('sanitizeFontFaces', () => {
     expect(sanitizeFontFamily("'Helvetica neue'")).toEqual("'Helvetica neue'")
   })
 
+  it('Several, improperly quoted', () => {
```

Comment:
Should this be "properly"?
```suggestion
  it('Several, properly quoted', () => {
```

Reply:
🤦  - yes, yes it should be. Fixed!